build.zig

```diff
@@ -55,8 +56,8 @@ pub fn build(b: &Builder) !void {
     addCppLib(b, exe, cmake_binary_dir, "zig_cpp");
     if (lld_include_dir.len != 0) {
         exe.addIncludeDir(lld_include_dir);
-        var it = mem.split(lld_libraries, ";");
-        while (it.next()) |lib| {
+        var it = try string.utf8Split(lld_libraries, ";");
```

Comment:
I need more clarification about what specifically you dislike :).  Is it the call name ?  Like is it just because it is not called `string.split` or is it the fact it uses generics behind the scene?

Again i'm happy to remove the type independent stuff if need be :)